ui/raidboss/data/05-shb/dungeon/twinning.txt

```diff
@@ -3,14 +3,10 @@
 hideall "--Reset--"
 hideall "--sync--"
 
-0.0 "--Reset--" sync / 00:0839::.*is no longer sealed/ window 100000 jump 0
-
-
 ### Alpha Zaghnal
 # -p 3D65:108.5 -ii 3D66 3D67 3D68 3D69 3D6A 41CA 3D6C
 100.0 "--sync--" sync / 00:0839::Repurposing will be sealed off/ window 100,0
-102.5 "--sync--" sync / 1[56]:[^:]*:Cladoselache:366:/ window 103,0
-108.5 "Augurium" sync / 1[56]:[^:]*:Alpha Zaghnal:3D65:/
+108.5 "Augurium" sync / 1[56]:[^:]*:Alpha Zaghnal:3D65:/ window 9,9
```

Comment:
```suggestion
108.5 "Augurium" sync / 1[56]:[^:]*:Alpha Zaghnal:3D65:/ window 109,9
```

This is maybe a workaround, but there's some large number of folks who have chat log messages turned off or the """for privacy""" ffxiv setting on.  The autos that were there before would have synced without the chat line.  Can you also do that here (and elsewhere)?

Reply:
> This is maybe a workaround, but there's some large number of folks who have chat log messages turned off or the """for privacy""" ffxiv setting on. The autos that were there before would have synced without the chat line. Can you also do that here (and elsewhere)?

I went back through and I think I got all of the instances in this PR where there were starts based on chatlog lines and added the correct (longer) sync on the first ability.  Some of it was inconsistent even in the same timeline (multiple-boss content like dungeons where the later bosses didn't have long syncs after the zone-seal msg).  And I tried to make it consistent even in content where there the first sync wasn't to an auto.  Definitely could use a close look though.